manifests/params.pp

```diff
@@ -67,7 +67,7 @@
   $email_from = "no-reply@${::domain}"
   $email_enabled = false
 
-  $lazy_enabled = false
+  $manage_squid = true
```

Comment:
Does it just continue to work as before if you use those settings but do not have squid?